nbpages/converter.py

```diff
@@ -409,18 +456,29 @@ def run_parsed(nbfile_or_path, output_type, args, **kwargs):
         raise IOError("Couldn't find template file at {0}"
                       .format(template_file))
 
+    if args.changed:
+        args.include = get_changed()
```

Comment:
This silently overwrites `args.include` if it was given by the user.  One could imagine complicated things to deal with this, but maybe the better thing to do is just:
```suggestion
        if args.include is not None:
            raise ValueError("cannot give an explicit include list and ask for only changed notebooks at the same time")
        args.include = get_changed()
```